modules/test/helpers/conditions.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2023 Red Hat
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	. "github.com/onsi/gomega"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/condition"
+	"k8s.io/apimachinery/pkg/types"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+type conditionsGetter interface {
+	GetConditions(name types.NamespacedName) condition.Conditions
+}
+
+type conditionGetterFunc func(name types.NamespacedName) condition.Conditions
+
+func (f conditionGetterFunc) GetConditions(name types.NamespacedName) condition.Conditions {
```

Comment:
There are still *unused* problems with those. Quite a few operators copy and paste these. Let's figure out how to properly plug that into the test helper

Reply:
I'm a bit lost with adapting those as proper interfaces after moved from nova tests